openpype/hosts/nuke/api/lib.py

```diff
@@ -3155,3 +3155,11 @@ def get_viewer_config_from_string(input_string):
             ).format(input_string))
 
     return (display, viewer)
+
+
+def is_headless():
+    """
+    Returns:
+        bool: headless
+    """
+    return QtWidgets.QApplication.instance() is None
```

Comment:
```suggestion
```
I suspect this can now be removed again?

Reply:
yeah, it had been solved in [`c414c8c` (#5107)](https://github.com/ynput/OpenPype/pull/5107/commits/c414c8c6e933e82e6e7703b8e509b711e36f9d2e)